### PR TITLE
nload: add new package

### PR DESCRIPTION
**Pull request by LXGMAX**
Signed-off-by: Jacky Guo <leickwell@Hotmail.com>

Maintainer: me
Compile tested: aarch64/mt7622), ramips/mt7628
Run tested: Redmi AX6S, Minew G1-C

Description:
A console application which monitors network
traffic and bandwidth usage in real time.
[https://github.com/rolandriegel/nload](https://github.com/rolandriegel/nload)

### CODE REVIEW

**Comment by LXGMAX** [comment]
Maintainer information was upgraded.

File: admin/nload/Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  URL:=https://github.com/rolandriegel/nload
+endef
+
+define Package/nload/description
+  monitors network traffic and bandwidth usage
+endef
+
+define Package/nload/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
```
Comment (1715173329) [inline admin/nload/Makefile:39]:
```suggestion
	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/nload $(1)/usr/bin/
```

Please squash all of your commits into a single one, and add a proper description & sign-off in your commit message.

**Comment by 1715173329** [comment]
I already told you that. idk why you marked it as "resolved".

**Comment by LXGMAX** [comment]
Sorry I'm new to contributing, is the current way of commit correct?

**Comment by 1715173329** [comment]
> Sorry I'm new to contributing, is the current way of commit correct?

```bash
git reset --soft cdd3b50a005131f6ad5e2be3e7979efbf9409074
git commit --amend -s
```

**Comment by LXGMAX** [comment]
Done.

**Pull request closed without merging**